pkg/cmd/connector/stop/stop.go

```diff
@@ -0,0 +1,91 @@
+package stop
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/connector/connectorcmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/contextutil"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	outputFormat string
+	f            *factory.Factory
+	connectorID  string
+}
+
+// NewStopCommand creates a new command to start a connector
+func NewStopCommand(f *factory.Factory) *cobra.Command {
+
+	opts := &options{
+		f: f,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "stop",
+		Short:   f.Localizer.MustLocalize("connector.stop.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("connector.stop.cmd.longDescription"),
+		Example: f.Localizer.MustLocalize("connector.stop.cmd.example"),
+		Hidden:  false,
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			validOutputFormats := flagutil.ValidOutputFormats
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, validOutputFormats...) {
+				return flagutil.InvalidValueError("output", opts.outputFormat, validOutputFormats...)
+			}
+
+			return runUpdateCommand(opts)
+		},
+	}
+
+	flags := connectorcmdutil.NewFlagSet(cmd, f)
+	flags.AddConnectorID(&opts.connectorID)
+	flags.AddOutput(&opts.outputFormat)
+
+	return cmd
+
+}
+
+func runUpdateCommand(opts *options) error {
+	f := opts.f
+
+	var conn connection.Connection
+	conn, err := f.Connection(connection.DefaultConfigSkipMasAuth)
+	if err != nil {
+		return err
+	}
+
+	if opts.connectorID == "" {
+		connectorInstance, instance_err := contextutil.GetCurrentConnectorInstance(&conn, f)
+		if instance_err != nil {
+			return instance_err
+		}
+
+		opts.connectorID = *connectorInstance.Id
+	}
+
+	api := conn.API()
+
+	patch := make(map[string]interface{})
+	patch["desired_state"] = "stopped"
```

Comment:
Replace "stopped" with enum from SDK

Reply:
This is big issue for SDKs. If you cannot effectively discover types in editor and github search doesn't work as well. 

See this: https://github.com/redhat-developer/app-services-sdk-go/blob/main/connectormgmt/apiv1/client/docs/ConnectorDesiredState.md

There is example how to use it etc.